webservices/docs.py

```diff
@@ -254,7 +254,9 @@
 take out items where `"memoed_subtotal":true`. Memoed items are subtotals of receipts
 that are already accounted for in another schedule a line item.
 
-For the Schedule A aggregates, "memoed" items are not included to avoid double counting.
+For the Schedule A aggregates such as `by_occupation` and `by_state` reflect individual
```

Comment:
How about "Schedule A aggregates, such as `by_occupation` and `by_state`, include only individual contributions, i.e. receipts from line numbers 11AI and 17A."